kolibri/plugins/learn/assets/src/views/LearnSideNavEntry.vue

```diff
@@ -14,20 +15,64 @@
   import CoreMenuOption from 'kolibri.coreVue.components.CoreMenuOption';
   import navComponents from 'kolibri.utils.navComponents';
   import urls from 'kolibri.urls';
+  import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
+  import { PageNames as LearnPageNames } from '../constants';
+  import { generateNavRoute } from '../../../../../core/assets/src/utils/generateNavRoutes';
+  import baseRoutes from '../routes/baseRoutes';
   import commonLearnStrings from './commonLearnStrings';
 
+  const url = urls['kolibri:kolibri.plugins.learn:learn']();
   const component = {
     name: 'LearnSideNavEntry',
-    mixins: [commonLearnStrings],
+    mixins: [commonLearnStrings, commonCoreStrings],
     components: {
       CoreMenuOption,
     },
     computed: {
-      url() {
-        return urls['kolibri:kolibri.plugins.learn:learn']();
+      learnRoutes() {
+        return {
+          home: {
+            text: this.coreString('homeLabel'),
+            icon: 'dashboard',
```

Comment:
My suggestion was to just add a boolean to these routes:
```
bottomBar: true,
```

Rather than define them all over again.